src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -760,10 +776,10 @@ private List<LaunchCommand> GetInitializeCommands()
                         commands.Add(new LaunchCommand("-environment-cd " + escapedDir));
                     }
 
-                    // TODO: The last clause for LLDB may need to be changed when we support LLDB on Linux
+                    // TODO: The last clause for LLDB may need to be changed when we support LLDB on Linux as LLDB's tty redirection doesn't work.
```

Comment:
Should this TODO statement be changed? I am assuming we actually can use LLDB on Linux now

Reply:
When we first invested time for 3.8.1 of LLDB I built and tested it on Linux and it "worked". I added the comment because the tty redirection doesn't seem to do anything because LLDB has a notion of lldb-launcher on Darwin only that launches a new terminal. It may be that LLDB on Linux with tty will work but I haven't rebuilt or tested it .